File: backend/src/main/java/at/ac/tuwien/sepm/groupphase/backend/endpoint/dto/location/LocationEventDto.java
package at.ac.tuwien.sepm.groupphase.backend.endpoint.dto.location;

import at.ac.tuwien.sepm.groupphase.backend.endpoint.dto.event.EventExpansionDetailsDto;

public record LocationEventDto(
    String name,
    String country,
    String town,
    String street,
    String postalCode,
    EventExpansionDetailsDto[] events) {}

File: frontend/src/app/dtos/location/location-event.dto.ts
import {EventExpansionDetailsDto} from '../event/event-expansion-details.dto';

export class LocationEventDto {
  constructor(
    public name: string,
    public street: string,
    public postalCode: number,
    public town: string,
    public country: string,
    public imageReference: string,
    public events: EventExpansionDetailsDto[]
  ) {
  }
}
